src/logger.js

```diff
@@ -80,6 +80,9 @@ class Logger {
       args.push((error, value) => {
         if (error) {
           logger.error(msg);
+          if (value) {
+            logger.error(value);
```

Comment:
This is in the `if (error)` case.

It's augmenting the error output with whatever happened to be in `value` when the error arose.